### PR TITLE
Restructure UI to make searching / typing a URL faster. (#832)

### PR DESCRIPTION
**Pull request by pocmo**
Changes:
* There's no HomeFragment anymore. Instead the URLInputFragment either has a background (gradient + logo) or it's translucent (shown on top of the browser)
* We start with the keyboard up and the cursor in the toolbar ready to start typing (also after erasing the session)

### CODE REVIEW

**Comment by pocmo** [comment]
@ekager Oops, I pushed code that doesn't compile. I just fixed that.

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 1]
```diff
@@ -663,7 +663,7 @@ public void eraseAndShowHomeScreen(final boolean animateErase) {
         }
 
         transaction
-                .replace(R.id.container, HomeFragment.create(), HomeFragment.FRAGMENT_TAG)
+                .replace(R.id.container,UrlInputFragment.createWithBackground(), UrlInputFragment.FRAGMENT_TAG)
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[Checkstyle\] ERROR: ',' is not followed by whitespace\.

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
-    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">"
```
Comment (ekager) [inline app/src/main/res/values/styles.xml]: Is this a rogue quotation mark?

Comment (pocmo) [inline app/src/main/res/values/styles.xml]: Oops 🔥

**Pull request merged**